suites/pacific/cephfs/tier-1_fs_kernel_pipeline.yaml

```diff
@@ -0,0 +1,230 @@
+                  label: mds
+          - config:
+              args:
+                - ceph
+                - fs
+                - set
+                - cephfs
+                - max_mds
+                - "2"
+              command: shell
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+        abort-on-fail: true
+        config:
+            command: add
+            id: client.1
+            node: node8
+            install_packages:
+                - ceph-common
+            copy_admin_keyring: true
+        desc: Configure the Cephfs client system 1
+        destroy-cluster: false
+        module: test_client.py
+        name: configure client
+  - test:
+        abort-on-fail: true
+        config:
+            command: add
+            id: client.2
+            node: node9
+            install_packages:
+                - ceph-common
+            copy_admin_keyring: true
+        desc: Configure the Cephfs client system 2
+        destroy-cluster: false
+        module: test_client.py
+        name: configure client
+  - test:
+        abort-on-fail: true
+        config:
+            command: add
+            id: client.3
+            node: node10
+            install_packages:
+                - ceph-common
+            copy_admin_keyring: true
+        desc: Configure the Cephfs client system 3
+        destroy-cluster: false
+        module: test_client.py
+        name: configure client
+  - test:
+        abort-on-fail: true
+        config:
+            command: add
+            id: client.4
+            node: node11
+            install_packages:
+                - ceph-common
+            copy_admin_keyring: true
+        desc: Configure the Cephfs client system 4
+        destroy-cluster: false
+        module: test_client.py
+        name: configure client
+  - test:
+      name: cephfs_volume_management
+      module: cephfs_volume_management.py
+      polarion-id: CEPH-83573445
+      desc: cephfs volume management
+      abort-on-fail: false
+  - test:
+      name: cephfs_snapshot_management
+      module: cephfs_snapshot_management.py
+      polarion-id: CEPH-83571366
+      desc: cephfs snapshot management
+      abort-on-fail: false
+  - test:
+        name: cephfs_tier1_ops
+        module: cephfs_tier1_ops.py
+        polarion-id: CEPH-83573447
+        desc: cephfs tier1 operations
+        abort-on-fail: false
+  - test:
+        name: cephfs_client_authorize
+        module: client_authorize.py
+        polarion-id: CEPH-83574483
+        desc: client authorize test for cephfs
+        abort-on-fail: false
+  - test:
+      name: cephfs-mdsfailover-pinning-io
+      module: dir_pinning.py
+      config:
+        num_of_dirs: 200
+      polarion-id: CEPH-11227
+      desc: MDSfailover on active-active mdss,performing client IOs with no pinning at the first,later pin 10 dirs with IOs
+      abort-on-fail: false
+  - test:
+      name: cephfs subvolume authorize test
```

Comment:
We are also running additional regression tests here that is already part other regression suites. Please help understand why we are repeating these tests in this suite?

Reply:
i agree with you. I will talk to Neha with details.

Reply:
Julius, As we discussed we need not repeat the tests in the kernel suite. Lets keep the exclusive tests only. The other tests are covered in regression runs.